src/frontend/src/common/components/library/data_subview/components/createTreeModal/components/RadioLabel/index.tsx

```diff
@@ -0,0 +1,80 @@
+// eslint-disable @typescript-eslint/explicit-member-accessibility
+import ListItemText from "@material-ui/core/ListItemText";
+import { List } from "czifui";
+import React from "react";
+import {
+  Label,
+  LabelLight,
+  LabelMain,
+  StyledDiv,
+  StyledIconCheckSmall,
+  StyledIconXSmall,
+  StyledListItem,
+  StyledListItemIcon,
+} from "./style";
+
+interface Props {
+  selected: boolean;
+}
+
+export const RadioLabelContextual = ({ selected }: Props): JSX.Element => {
+  return (
+    <StyledDiv>
+      <Label>
+        <LabelMain>My county and other regions </LabelMain>
+        <LabelLight>- Recommended</LabelLight>
+      </Label>
+      {selected && (
+        <List>
+          <StyledListItem button={false as any}>
+            <StyledListItemIcon>
+              <StyledIconCheckSmall />
+            </StyledListItemIcon>
+            <ListItemText>Best for local outbreak investigation.</ListItemText>
+          </StyledListItem>
+          <StyledListItem button={false as any}>
+            <StyledListItemIcon>
+              <StyledIconCheckSmall />
+            </StyledListItemIcon>
+            <ListItemText>
+              Best for seeing relationships between your selected samples, other{" "}
+              samples you have uploaded to Aspen, and publicly-available samples
+              on GISAID.
+            </ListItemText>
+          </StyledListItem>
+        </List>
+      )}
+    </StyledDiv>
+  );
+};
+
+export const RadioLabelLocal = ({ selected }: Props): JSX.Element => {
+  return (
+    <StyledDiv>
+      <LabelMain>My county only </LabelMain>
+      {selected && (
+        <List>
+          <StyledListItem button={false as any}>
```

Comment:
Why have you added `as any` for the typing here? Does typescript fail to compile because you're trying to make it a styled component?

Reply:
oh i thought i added a comment for this. apparently you need to pass in the button prop otherwise typescript complains, and the button props type is set to true (weird) so i couldn't pass in false. Do you know of a better way to do this?